app/services/lom_service/export_lom.rb

```diff
@@ -38,7 +38,9 @@ def oml_general(xml)
         end
         xml.language @task.iso639_lang
         xml.description do
-          xml.string ApplicationController.helpers.render_markdown(@task.description), language: @task.iso639_lang
+          html_fragment = Loofah.fragment(ApplicationController.helpers.render_markdown(@task.description))
+          html_fragment.scrub!(NbpScrubber.new)
```

Comment:
Here (and in the `NbpScrubber`) you are (implicitly) using two dependencies: `loofah` and `rails-html-sanitizer`. 

Just thinking aloud for a moment: Those two requirements are not explicitly listed (yet) in our `Gemfile` (only `Gemfile.lock`). The code currently works, since both are included implicitly through `actionpack` and `actionview`, which are dependencies for `rails`. If Rails switches to another library (or, for example drops the usage of `loofah`, which is not part of github.com/rails), our code will break. Hence, I was wondering whether we should list one (`rails-html-sanitizer`) or both (`rails-html-sanitizer`, `loofah`) dependencies directly in our `Gemfile.lock`).

What do you think?

Reply:
Yes, I added both.